timetracker/entries/urls.py

```diff
@@ -2,11 +2,14 @@
 
 from . import views
 
+from .models import Client
+from .views import ClientCreateView, ProjectCreateView, RootRedirectView
 
 urlpatterns = [
-    url(r'^clients/$', views.clients, name='client-list'),
+	url(r'^$', RootRedirectView.as_view(), name='root'),
```

Comment:
You should be careful with using tabs and spaces for indentation. Recommendation is to use 4 spaces.